Sources/SectionItemsProvider/LazySectionItemsProvider.swift

```diff
@@ -2,9 +2,16 @@
 //  Copyright © 2020 Rosberry. All rights reserved.
 //
 
-import UIKit
-
-public final class LazySectionItemsWrapper {
+/// It  is an implementation of `SectionItemsProvider` that organizes an access to required
+/// cellItems and sectionItems on demand. It stores section created items in dictionary and calls
```

Comment:
I don't get what "It stores section created items" means 🤔

Reply:
fixed to `It stores created section items in`